eng/scripts/SetupPostAnalysisVariables.ps1

```diff
@@ -0,0 +1,20 @@
+[CmdletBinding()]
```

Comment:
Is it worth having this be a script? It might be better just to do this inline in the yml file.